siliconcompiler/report/streamlit_viewer.py

```diff
@@ -632,10 +635,15 @@ def show_title_and_runs(title_col_width=0.7):
     metrics_tab, manifest_tab, file_preview_tab = tabs
 
 with metrics_tab:
+    if ui_width <= 0:
+        # in case streamlit_javascript errors
+        ui_width = 1
```

Comment:
What does this default translate to in terms of looks?

Reply:
it might be better to not adjust this number and simply break up the calculations as:
```
width1 = 0.4
width2 = 0.1
if ui_width > 0:
  width1 = min(width1, .../ui_width)
  ...
```